test/testlib/minikube_utilities.go

```diff
@@ -1313,6 +1313,12 @@ func ScaleStatefulSet(t *testing.T, namespaceName string, name string, replicas
 	k8s.RunKubectl(t, options, "scale", "statefulset", name, fmt.Sprintf("--replicas=%d", replicas))
 }
 
+func ScaleDeployment(t *testing.T, namespaceName string, name string, replicas int) {
```

Comment:
Nit: variable `name` is a bit vague for someone trying to understand the function from the signature. Maybe rename it to `deploymentName`?

Reply:
I presumed that as the function is called `ScaleDeployment`, then it is clear that the parameter `name` corresponds to the deployment name. The `ScaleStatefulSet` is having the same signature.